### ISSUE TITLE
Install step-ca binary >=0.23: mv: cannot stat ''/tmp/step-ca_0.25.2/step-ca'': No such file or directory

### ISSUE THREAD

**Issue by patsevanton** [maxhoesel-ansible/ansible-collection-smallstep#389]
Hello! Thanks for ansible collection
i try ca init and get error:

Install
```
ansible-galaxy collection install maxhoesel.smallstep>=0.25.2,0.26.0
ansible-playbook -i inventory.yml ca.yml
```

My ca.yaml
```
- hosts: step_ca
  become: yes
  tasks:
    # Install and initialize the CA server.
    # There are a lot of configuration options, see the step_ca README for details
    - name: Install step-ca
      include_role:
        name: maxhoesel.smallstep.step_ca
      vars:
        step_ca_name: Foobar Internal CA
        step_ca_root_password: "incredibly secret password"
        step_ca_intermediate_password: "very secret password"
```

Inventory:
```
all:
  children:
    freipa-ca:
      hosts:
        "step_ca":
          ansible_host: "158.160.67.97"
  vars:
    ansible_user:  ubuntu
    ansible_ssh_private_key_file: ~/.ssh/id_rsa
```

Get error:
```
TASK [maxhoesel.smallstep.step_ca : Install step-ca binary >=0.23] *******************************************************************************************
fatal: [step_ca]: FAILED! => changed=true 
  cmd: |-
    set -o pipefail && mv -Z /tmp/step-ca_0.25.2/step-ca /usr/bin
  delta: '0:00:00.003487'
  end: '2024-03-02 06:59:17.171144'
  msg: non-zero return code
  rc: 1
  start: '2024-03-02 06:59:17.167657'
  stderr: 'mv: cannot stat ''/tmp/step-ca_0.25.2/step-ca'': No such file or directory'
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>
```

list tmp
```
root@smallstep:~# ls -1 /tmp/
LICENSE
README.md
step-ca
systemd-private-94af797284844435a060a75d2ce64242-systemd-logind.service-DrENPh
systemd-private-94af797284844435a060a75d2ce64242-systemd-resolved.service-K9K9ai
systemd-private-94af797284844435a060a75d2ce64242-systemd-timesyncd.service-lz7OGh
terraform_701826087.sh
```

echo step_ca_version
```
TASK [maxhoesel.smallstep.step_ca : Set latest release version] **********************************************************************************************
task path: /home/user/.ansible/collections/ansible_collections/maxhoesel/smallstep/roles/step_ca/tasks/main.yml:17
ok: [step_ca] => changed=false 
  ansible_facts:
    step_ca_version: 0.25.2
```

TASK [maxhoesel.smallstep.step_ca : Download and extract step-ca archive] ************************************************************************************
```
changed: [step_ca] => changed=true 
  dest: /tmp/
  extract_results:
    cmd:
    - /usr/bin/tar
    - --extract
    - -C
    - /tmp/
    - -z
    - -f
    - /home/ubuntu/.ansible/tmp/ansible-tmp-1709363102.0097518-9511-210232244944279/step-ca_linux_0.25.2_amd6413ohfeko.tar.gz
    err: ''
    out: ''
    rc: 0
  gid: 0
  group: root
  handler: TgzArchive
  invocation:
    module_args:
      attributes: null
      copy: true
      creates: null
      decrypt: true
      dest: /tmp/
      exclude: []
      extra_opts: []
      group: null
      include: []
      io_buffer_size: 65536
      keep_newer: false
      list_files: false
      mode: null
      owner: null
      remote_src: true
      selevel: null
      serole: null
      setype: null
      seuser: null
      src: https://github.com/smallstep/certificates/releases/download/v0.25.2/step-ca_linux_0.25.2_amd64.tar.gz
      unsafe_writes: false
      validate_certs: true
  mode: '01777'
  owner: root
  size: 4096
  src: /home/ubuntu/.ansible/tmp/ansible-tmp-1709363102.0097518-9511-210232244944279/step-ca_linux_0.25.2_amd6413ohfeko.tar.gz
  state: directory
  uid: 0

```

**Comment by patsevanton** [comment]
I think it's time to create a new version.

**Comment by maxhoesel** [comment]
> ansible-galaxy collection install maxhoesel.smallstep>=0.25.2,0.26.0

is this command working for you? There is currently no 0.25 release of this collection, I am working on migrating to a new release structure (see #335), but that will take some more time. In either case, the issue you are describing was fixed in #363, but that PR has not landed in any release yet. You could try using a specific github commit as your collection version instead of the galaxy release for now (see here: https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-multiple-roles-from-a-file)

Closing as resolved in #363
